src/javascript/index.js

```diff
@@ -0,0 +1,16 @@
+import 'bootstrap/dist/css/bootstrap.min.css';
+import 'bootstrap';
+import '../css/styles.css';
+import '../css/switchStyle.css';
+import '../css/loader.css';
+import '@fortawesome/fontawesome-free/js/fontawesome';
+import '@fortawesome/fontawesome-free/js/solid';
+import '@fortawesome/fontawesome-free/js/regular';
+import '@fortawesome/fontawesome-free/js/brands';
+import home from './view/home';
+import populateMain from './controller/weatherControler';
+
+home();
+
+populateMain();
+// fetchWeatherData();
```

Comment:
### Remove commented out code
- [ ] It’s better to delete **commented piece of codes**. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))

Reply:
Done.